apiserver/facades/controller/caasapplicationprovisioner/provisioner.go

```diff
@@ -872,6 +918,7 @@ func (a *API) UpdateApplicationsUnits(ctx context.Context, args params.UpdateApp
 			result.Results[i].Error = apiservererrors.ServerError(err)
 			continue
 		}
+		//
```

Comment:
Delete me?

Reply:
ah, that was to remind me to add a card to deal with the things on app besides life in this place